retail-managment-system/rms-ui/src/app/shared/components/loading-spinner/loading-spinner.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+import { LoaderService } from 'src/app/core/services/loader.service';
+
+@Component({
+  selector: 'app-loading-spinner',
+  templateUrl: './loading-spinner.component.html',
+  styleUrls: ['./loading-spinner.component.scss']
+})
+export class LoadingSpinnerComponent implements OnInit {
+
+  constructor(public loader: LoaderService) { }
+
+  ngOnInit(): void {
```

Comment:
if you do not use it, you can delete it

Reply:
and also scss file

Reply:
deleted scss file
but needed loader t o use set and get methods in service file within html file

Reply:
here I mean onint function only not constructor

Reply:
done